resources/windows/wix/customactions.wxs

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="windows-1252"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <ComponentGroup Id="Include_CustomActions" />
+
+    <CustomAction Id="SaveCmdLineInstallDir"
+      Property="CMDLINE_INSTALLDIR"
+      Value="[INSTALLDIR]"
+      Execute="firstSequence" />
+    <CustomAction Id="SetFromCmdLineInstallDir"
+      Property="INSTALLDIR"
+      Value="[CMDLINE_INSTALLDIR]"
+      Execute="firstSequence" />
```

Comment:
Wouldn't that make my approach better from a users perspective? If there are sequence specific custom actions, they can be specified in the project. But any default actions will already be there and if there are no custom actions, then the user doesn't need to add anything additional at all. If we had a `customactions-common.wxs` file in vanagon, would we have to add these user-specified variables to that file, or would they be seperate wxs files? How do you guarentee order if they are seperate files?